ui/admin/app/templates/scopes/scope/workers/index.hbs

```diff
@@ -145,6 +149,58 @@
           </B.Tr>
         </:body>
       </Hds::Table>
+
+      {{#if this.selectedWorker}}
+        <Hds::Flyout
+          data-test-worker-tags-flyout
+          @onClose={{fn this.selectWorker null}}
+          as |M|
+        >
+          <M.Header>
+            {{t
+              'resources.worker.flyout.title'
+              workerName=this.selectedWorker.displayName
+            }}
+          </M.Header>
+          <M.Body>
+            <Hds::Table
+              @model={{this.selectedWorkerTags}}
+              @columns={{array
+                (hash label=(t 'resources.worker.flyout.table.tag'))
+                (hash label=(t 'resources.worker.flyout.table.type'))
+              }}
+              @valign='middle'
+            >
+              <:body as |B|>
+                <B.Tr>
+                  <B.Td>
+                    <Hds::Text::Code class='tag-container'>
+                      <div class='tag-shortener'>
```

Comment:
Was this div actually necessary? Couldn't we have just had the text in the `Hds::Text::Code`?

Reply:
Nope not needed. I originally used a different approach that needed the div but changed it and didn't go back and remove it. Updated.